### PR TITLE
docs: add chart field to application.yaml

### PR DESCRIPTION
**Pull request by crenshaw-dev**
I thought it was weird that this field wasn't in the example yaml. I'm not 100% certain that my comment is correct/complete. Can someone please confirm that I'm correct before merging?

### CODE REVIEW

File: docs/operator-manual/application.yaml [hunk 1]
```diff
@@ -18,6 +18,7 @@ spec:
     path: guestbook
 
     # helm specific config
+    chart: chart-name  # Set this when pulling directly from a Helm repo. DO NOT set for git-hosted Helm charts.
```
Comment (jannfis) [inline docs/operator-manual/application.yaml:21]: :+1: 

I think the `source` part of the application requires even more clarification for Helm. For example, we should document that `repoURL` can be either pointing to a Helm chart repository or to a Git repository. `targetRevision` also has a slightly different semantic between Helm and Git repositories.

Comment (crenshaw-dev) [reply docs/operator-manual/application.yaml:21]: @jannfis thoughts on the comments I've added?

Comment (jannfis) [reply docs/operator-manual/application.yaml:21]: Thanks! LGTM.

**Review by jannfis — APPROVED** [review]
LGTM

**Pull request merged**